test/fixtures_missed_call.js

```diff
@@ -0,0 +1,204 @@
+// identities
+// +2345059991111 - Existing identity with active subscription
+// +2345059992222 - Existing identity with no subscription
+// +2345059993333 - No identity or subscription
```

Comment:
It might be helpful to export these phone numbers as constants so that we have a consistent reference to the identity across our test files and avoids any potential sanfus with typos.
e.g.
`const IDENTITY_WITH_ACTIVE_SUBSCRIPTION_MSISDN = '+2345059991111'`

Reply:
Thanks @nathanbegbie , I have added them as normal variables and it's working, jshint was not happy about the `export const` part. Let me know what you think.